### PR TITLE
create ui for favorite screen

### PR DESCRIPTION
**Pull request by anhmv-1205**
#### Screenshots
p21

### CODE REVIEW

**Comment by anhmv-1205** [comment]
ready

File: app/src/main/java/com/sun_asterisk/moviedb_44/screen/favorite/FavoriteFragment.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            MovieLocalDataSource.getInstance(MovieDatabase.getInstance(context!!).movieDAO()),
+            MovieRemoteDataSource.getInstance())
+        viewModel = FavoriteViewModel(movieRepository, this)
+        binding.viewModel = viewModel
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        viewModel.onStart()
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/moviedb_44/screen/favorite/FavoriteFragment.kt:36]: chạy trong onStart()

File: app/src/main/java/com/sun_asterisk/moviedb_44/screen/main/MainActivity.kt [hunk 2]
```diff
@@ -6,25 +6,48 @@ import androidx.appcompat.app.AppCompatActivity
...
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
         initViews()
     }
 
     private fun initViews() {
-        addFragment(HomeFragment(), R.id.container)
+        favoriteFragment = FavoriteFragment()
+        addFragment(favoriteFragment, R.id.container)
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/moviedb_44/screen/main/MainActivity.kt]:
```suggestion
        addFragment(FavoriteFragment().getInstance(), R.id.container)
```

File: app/src/main/java/com/sun_asterisk/moviedb_44/screen/main/MainActivity.kt [hunk 3]
```diff
@@ -6,25 +6,48 @@ import androidx.appcompat.app.AppCompatActivity
...
+        homeFragment = HomeFragment()
+        addFragment(homeFragment, R.id.container)
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/moviedb_44/screen/main/MainActivity.kt]:
```suggestion
        addFragment(HomeFragment(), R.id.container)
```

**Comment by anhmv-1205** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**